src/stores/store-urls-custom-admin.md

```diff
@@ -87,6 +90,21 @@ As a precaution, do not try to change the Admin URL yourself unless you know how
    {:.bs-callout-info}
    Note that the changes made using the Method 1 will have the priority over the changes made in the `app/etc/env.php` file.
 
+## Method 3: Change the Admin Path via CLI
+
+Magento provides a command to change the Admin Path - `setup:config:set`.
+
+1. To change the admin `frontName`, from the Magento root, run:
+
+    ```terminal
+    bin/magento setup:config:set --backend-frontname="backend_front_name"
+    ```
+
+    where `backend_front_name` is the new Admin Path.
+
+    {:.bs-callout-info}
+    This command will update `backend` > `frontName` configuration option in the `app/etc/env.php` file.
```

Comment:
I consolidated the information, so this can be cleaned up a bit.

```suggestion
This command updates the `backend` > `frontName` configuration option in the `app/etc/env.php` file.
```